crowdsourcing/viewsets/csvmanager.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework.viewsets import ViewSet
+from rest_framework.response import Response
+import csv
+
+class CSVManagerViewSet(ViewSet):
+
+    def parse(self, request):
+        uploadedFile = request.data['file']
+        csvinput = csv.DictReader(uploadedFile)
+        arr = []
+        for row in csvinput:
+            arr.append(row)
+        return Response(arr)
```

Comment:
great that you moved this to it's own class so that we can reuse it
we should improve this because the data is doing a round trip and another POST that is: it's being uploaded, processed, returned back to the client and uploaded again (pretty much), you see the problem right?
an idea would be you upload the file once, process the metadata only (store the file, I can provide you a model for now) and return only the metadata which means column names and number of rows and a file id
now you are not returning the data anymore, and you are not resending the data to the server again because we already have it.
@ksetyadi  @neilthemathguy  your input please, thank you